etlman/projects/admin.py

```diff
@@ -17,7 +17,7 @@ class StepInline(admin.TabularInline):
 
 class ProjectAdmin(admin.ModelAdmin):
     inlines = [CollaboratorInline, DataInterfaceInLine]
-    list_display = ["name", "description"]
+    list_display = ["id", "name", "description"]
```

Comment:
Nitpick (ID becomes the link and is very small to click on):
```suggestion
    list_display = ["name", "description"]
```

(If there is more to this than I'm getting please push back!)

Reply:
I included them because prior to implementing link buttons, I wanted to see the id of the object. I am fine with the removal.